### PR TITLE
Add ESPnet2 LJSpeech recipe

### PR DESCRIPTION
**Pull request by kan-bayashi**
Add espnet2 ljspeech recipe.

## TTS Template modification

- Add `g2p_en_no_space` phoneme tokenizer
- Use `feats_type=raw` as default
- Change the default `tag`, `token_list`, and `tts_stats_dir` naming to depend on the cleaner and g2p type [`4a68da5` (#2117)](https://github.com/espnet/espnet/pull/2117/commits/4a68da570ca28737093453f8d17afd876240afa1)

### CODE REVIEW

File: egs2/TEMPLATE/asr1/db.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@ if [[ "$(hostname -d)" == clsp.jhu.edu ]]; then
     HKUST1=
     HKUST2=
     LIBRISPEECH=
+    LJSPEECH=
```
Comment (sw005320) [inline egs2/TEMPLATE/asr1/db.sh:23]: I think it’s better to move `db.sh` under the `TEMPLATES directory (or some common directory for both asr and tts recipes) or prepare it in both `asr1` and `tts1`.

Comment (kan-bayashi) [reply egs2/TEMPLATE/asr1/db.sh:23]: What does it mean?
You mean we should prepare two db.sh for tts and asr?

Comment (sw005320) [reply egs2/TEMPLATE/asr1/db.sh:23]: sorry for the confusion. please ignore my comment.

**Comment by kan-bayashi** [comment]
Finished.
@kamo-naoyuki Could you take a look at the template modification?

File: egs2/ljspeech/tts1/local/data.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+db_root=${LJSPEECH}
+
```
Comment (kamo-naoyuki) [inline egs2/ljspeech/tts1/local/data.sh:29]: Can add checking the value of LJSPEECH?
(I also forgot doing it in jsut)

https://github.com/espnet/espnet/blob/c997249cf96b803bb7293d9fbda6905a56ed9f93/egs2/commonvoice/asr1/local/data.sh#L25-L28

Comment (kan-bayashi) [reply egs2/ljspeech/tts1/local/data.sh:29]: Sure!

Comment (kan-bayashi) [reply egs2/ljspeech/tts1/local/data.sh:33]: Since we will download the corpus if not exists, `-z` is better?

Comment (kamo-naoyuki) [reply egs2/ljspeech/tts1/local/data.sh:33]: Oh, yes, you are right.

**Comment by kan-bayashi** [comment]
I will merge.

**Pull request merged**